src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -240,3 +245,128 @@ def reshape_to_vivarium_format(df, location):
+    # Prevalence of BBBM state among simulants in any state
+    all_state_dur = load_alzheimers_duration_all_states(None, location, years)
+    return data_values.BBBM_AVG_DURATION / all_state_dur
+
+
+def load_mci_conditional_prevalence(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    # Prevalence of MCI state among simulants in any state
+    all_state_dur = load_alzheimers_duration_all_states(None, location, years)
+    return data_values.MCI_AVG_DURATION / all_state_dur
+
+
+def load_bbbm_incidence_count(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    DUR = 7  # total duration of pre-dementia AD
+    W = 5  # width of age groups
+    R = 2  # remainder (DUR = nw + r)
+
+    inc = get_data(data_keys.ALZHEIMERS.INCIDENCE_RATE, location, years)
+    pop = get_data(data_keys.POPULATION.STRUCTURE, location, years)
+    new_bbbm_people = pd.DataFrame(0, index=pop.index, columns=pop.columns)
+    for index, row in new_bbbm_people.iterrows():
+        # create tuples to index population for future age groups and year
+        next_group_future_year = tuple(map(add, list(index), ["", "", W, W, DUR, DUR]))
+        scnd_group_future_year = tuple(
+            map(add, list(index), ["", "", 2 * W, 2 * W, DUR, DUR])
+        )
+        # cap age and year
+        age_end = min(100, next_group_future_year[3])
+        next_group_future_year = next_group_future_year[:2] + (
+            min(95, next_group_future_year[2]),
+            age_end if age_end < 100 else 125,
+            min(2050, next_group_future_year[4]),
+            min(2051, next_group_future_year[5]),
+        )
+        age_end = min(100, scnd_group_future_year[3])
+        scnd_group_future_year = scnd_group_future_year[:2] + (
+            min(95, scnd_group_future_year[2]),
+            age_end if age_end < 100 else 125,
+            min(2050, scnd_group_future_year[4]),
+            min(2051, scnd_group_future_year[5]),
+        )
+        # get age group incidences
+        # TBD should new_bbbm_people use inc draws? currently single value
+        next_group_inc = inc.loc[next_group_future_year[1:4] + (2021, 2022)].mean()
+        scnd_group_inc = inc.loc[scnd_group_future_year[1:4] + (2021, 2022)].mean()
+
+        # set value
+        # pdb.set_trace()
+        # print([next_group_inc, pop.loc[next_group_future_year].value, scnd_group_inc, pop.loc[scnd_group_future_year].value])
+        new_bbbm_people.loc[index] = (
+            (1 - (R / W)) * next_group_inc * pop.loc[next_group_future_year].value
+        ) + ((R / W) * scnd_group_inc * pop.loc[scnd_group_future_year].value)
+    # TBD add deaths and scale
+    # pdb.set_trace()
+    return new_bbbm_people
+
+
+def load_mci_disability_weight(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    data = pd.read_csv("/ihme/epi/disability_weights/standard/dw_full.csv")
```

Comment:
perhaps we should make a copy of this csv, so that we know we can always find it